gui/slick/interfaces/default/home.mako

```diff
@@ -222,7 +224,7 @@ $(document).ready(function(){
             4: { sorter: 'quality' },
             5: { sorter: 'eps' },
             % if sickbeard.FILTER_ROW:
-            6: { filter : 'parsed' }
+                7: { filter : 'parsed' }
             % endif
         },
```

Comment:
hmm, I thought we could change column order before. either I am mistaken or we lost that ability somehow.

what are the values in `sortList: [[6,1],[2,0]],` for?